mono/mini/TestHelpers.cs

```diff
@@ -0,0 +1,31 @@
+using System;
```

Comment:
Given that this is used in the corlib tests, I think it would be better to store the file in `mcs/class/test-helpers` for consistency.